jaxws-ri/tools/wscompile/src/test/java/com/sun/tools/ws/test/processor/modeler/annotation/NillableTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Distribution License v. 1.0, which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+package com.sun.tools.ws.test.processor.modeler.annotation;
+
+import javax.jws.WebService;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlElementWrapper;
+
+@WebService()
+public class NillableTest {

```

Comment:
this is not a test, so the name must not end with "Test" suffix otherwise tools (maven) will treat this as a runnable test and will act as such. It is also confusing ...